src/applications/static-pages/health-care-manage-benefits/constants.js

```diff
@@ -0,0 +1 @@
+export const CERNER_FACILITY_IDS = ['668', '757'];
```

Comment:
lmk if you know of a better place to put this, I thought this would be fine for now as _hopefully_ this is just a temporary thing lol